apps/antalmanac/src/components/RightPane/SectionTable/EnrollmentHistoryPopup.tsx

```diff
@@ -64,32 +78,55 @@ export function EnrollmentHistoryPopup({ department, courseNumber }: EnrollmentH
         );
     }
 
+    const handleBack = () => {
+        setGraphIndex((prev) => prev - 1);
+    };
+
+    const handleForward = () => {
+        setGraphIndex((prev) => prev + 1);
+    };
```

Comment:
You could memoize these callbacks if you want.

```suggestion
    const handleBack = useCallback(() => {
        setGraphIndex((prev) => prev - 1);
    }, []);

    const handleForward = useCallback(() => {
        setGraphIndex((prev) => prev + 1);
    }, []);
```